### DISCUSSION TITLE
[BP#6] Privilégier une approche "mobile first", à défaut RESS

### DISCUSSION THREAD

**Discussion by tbroyer** [cnumr/best-practices#51] [Bonnes pratiques] [answered]
Plusieurs choses sur cette BP:

* RESS.io est mort depuis au moins août 2015. cf. https://web.archive.org/web/20150701000000*/http://ress.io/
* RWD et RESS ne sont pas des termes qu'on voit beaucoup employés
* Est-ce qu'on ne devrait pas expliciter du coup ces approches à base de media queries et srcsets, voire un peu de JS (RWD) vs Client Hints (RESS)

Resources:
* https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/client-hints
* https://developers.google.com/web/updates/2015/09/automating-resource-selection-with-client-hints
* https://web.dev/image-cdns/

**Comment by ACTLEM** [comment]
Effectivement on entend plus parler de Responsive Design plutôt que de l'acronyme RWD. Pour le RESS, on parle plus de "mixer" du Responsive Design avec du SSR.
C'est un bon point de rendre le vocabulaire de cette BP plus proche de celui du terrain. Mettre des exemples concrets (media query et client hints pourraient suffire) permettraient de clarifier le propos sachant que l'essentiel est le "mobile first" (qui lui est compris par tout le monde).

**Comment by rlemaire** [comment]
Je pense que la mention de Responsive Web Design Server Side est à retirer. Je ne l'ai jamais rencontré sur le terrain, même quand j'en ai fait à PhoneValley (agence à l'époque spécialisé dans les sites mobiles, dont wap, blackberry, etc...).

Je serais pour parler des outils des navigateurs pour sélectionner du contenu : media query, balise, picture, ...

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/142
